examples/src/test/scala/docs/eventsourced/ShoppingCart.scala

```diff
@@ -130,17 +134,20 @@ object ShoppingCart {
 
   final case class CheckedOut(cartId: String, eventTime: Instant) extends Event
 
+  //#shardingTags
   val EntityKey: EntityTypeKey[Command] = EntityTypeKey[Command]("ShoppingCart")
+  val tags = Set("carts-0", "carts-1", "carts-2", "carts-3", "carts-4").toIndexedSeq
```

Comment:
```
val tags = Vector("carts-0", "carts-1", "carts-2", "carts-3", "carts-4")
```

otherwise the order is not guaranteed and daemon `n` might not correspond to `carts-n`, and maybe not even deterministic

Reply:
Oh yeah, of course